quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-with-nat-gateway/azuredeploy.json

```diff
@@ -192,6 +193,7 @@
               "networkSecurityGroup": {
                 "id": "[variables('nsgId')]"
               },
+              "defaultOutboundAccess": false,
               "natGateway": {
```

Comment:
for workspace resource if we have depends on vNet that is sufficient, looks like old code has issue fix it

"dependsOn": [
        "[resourceId('Microsoft.Network/networkSecurityGroups', parameters('nsgName'))]",
        "[resourceId('Microsoft.Network/virtualNetworks', parameters('vnetName'))]"
      ]